packages/terra-framework-docs/src/terra-dev-site/doc/data-grid/WorklistDataGrid.1/Examples.3/RowSelection.3.doc.mdx

```diff
@@ -1,3 +1,39 @@
 import RowSelection from './RowSelection?dev-site-example';
 
-<RowSelection title='Worklist Data Grid With Row Selection' />
+# Row Selection
+
+### Description
+The [Worklist DataGrid](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/about) is built to operate in either of two selection modes:
+* Cell Selection (_default_)
+* **Row Selection**
+
+Row Selection Mode is controlled via a prop and is off by default. When creating a Worklist DataGrid that supports Row Selection, the prop has to be explicitly set by the Consumer. This example demonstrates a Worklist DataGrid with full support for row selection and can be referenced for [row centric functionalities](http://localhost:8080/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/examples/row-selection#implemented-functionality).
```

Comment:
Why is the localhost url referenced here?

Reply:
It was by mistake. Removed.